packages/client/hmi-client/src/components/widgets/tera-pdf-panel.vue

```diff
@@ -24,11 +24,16 @@ interface PdfData {
 	name: string;
 }
 
+const emit = defineEmits(['pdf-ref']);
+
 const props = defineProps<{
 	pdfs: PdfData[];
 }>();
 
 const pdfs = ref<PdfData[]>(props.pdfs);
+const pdfRef = ref(null);
+
+onMounted(() => emit('pdf-ref', pdfRef));
```

Comment:
Probably @shawnyama or @blanchco can comment on this, I think we are using "defineExpose" generally, for exposing children-components to that of their parents, but I am not 100% sure, or if there are any trade offs between emit vs expose.

Reply:
ya looking at this `defineExpose` would be a cleaner solution